jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/LoadDockerStep.java

```diff
@@ -100,7 +100,12 @@ private BuildResult afterPushBaseImageLayerFuturesFuture()
           buildConfiguration.getTargetImageConfiguration().getImage();
 
       // Load the image to docker daemon.
-      dockerClient.load(new ImageToTarballTranslator(image).toTarballBlob(targetImageReference));
+      buildConfiguration
+          .getEventDispatcher()
+          .dispatch(
+              LogEvent.debug(
```

Comment:
Just curious: on a command line, the actual "docker load" outputs dynamic progress bars. I wonder if this will clutter up the debug log a lot.

Reply:
Just tried it, and it looks like there's very little output added.

Reply:
It may be that layers are already cached. I see when all the layers are cached, it just prints one line: 

First time:
```
$ docker load < target/jib-image.tar
c09300da4550: Loading layer [==================================================>]  7.859MB/7.859MB
72e3aebff08b: Loading layer [==================================================>]  598.6kB/598.6kB
6189abe095d5: Loading layer [==================================================>]  643.7kB/643.7kB
25d507fd9f9c: Loading layer [==================================================>]  38.89MB/38.89MB
6069c8819832: Loading layer [==================================================>]  2.477MB/2.477MB
a56689a3c89a: Loading layer [==================================================>]     164B/164B
d20deb22dd72: Loading layer [==================================================>]     895B/895B
547c0f0f2e68: Loading layer [==================================================>]  1.233kB/1.233kB
Loaded image: francium25/test:latest
```
Second time:
```
$ docker load < target/jib-image.tar
```

Reply:
Tried clearing the cache and docker daemon, still seems to be very little output caused by `docker load`. So I think we're good here (unless there's something I need to delete that I'm not finding).